traffic_ops/traffic_ops_golang/deliveryservicesregexes/deliveryservicesregexes.go

```diff
@@ -332,6 +342,18 @@ func validateDSRegexType(tx *sql.Tx, typeID int) error {
 	return err
 }
 
+func validateDSRegexOrder(tx *sql.Tx, dsID int, order int) error {
```

Comment:
looks like i can create a regex with order < 0. want to fix that?

Reply:
Fixed now. TP stops it by default, so I missed changing it in the API.